.meshrc.yaml

```diff
@@ -155,6 +155,12 @@ sources:
             method: GET
             responseSchema: ./json-schemas/zipLink.json
             responseTypeName: ZipLink
+          - type: Query
+            field: Run
```

Comment:
is this the correct value for `field` / `responseTypeName`?

Reply:
yeah, it is called Run in next gen.